alf/bin/play.py

```diff
@@ -42,7 +42,7 @@
     'checkpoint_name', None, "name of the checkpoint "
     "(e.g. 'ckpt-12800`). If None, the latest checkpoint under train_dir will "
     "be used.")
-flags.DEFINE_bool('greedy_predict', True, "use greedy action for evaluation.")
+flags.DEFINE_bool('epsilon_greedy', 0.1, "probability of sampling action.")
```

Comment:
DEFINE_bool -> DEFINE_float, and the default value should be 0?

Reply:
Good catch. Changed to DEFINE_float. The greedy_predict=True means epsilon_greedy=0.1. greedy_predict=1 means espsilon_greedy=1.0